client/src/ScatterPlot.js

```diff
@@ -0,0 +1,77 @@
+import React, {useRef, useEffect} from 'react'
+import {
+  select,
+  scaleLinear,
+  scaleTime,
+  min,
+  max,
+  axisBottom,
+  axisLeft,
+  timeWeek,
+} from 'd3'
+import './ScatterPlot.css'
+
+export default ({data}) => {
+  const container = useRef(null)
+
+  useEffect(() => {
+    const width = 1250
+    const height = 750
+    const padding = 50
+
+    const svg = select(container.current)
+    svg.attr('width', width).attr('height', height)
+
+    const dates = data.map((d) => d[0])
+    const xScale = scaleTime()
+      .domain([min(dates), max(dates)])
+      .range([padding, width - padding])
+
+    const yScale = scaleLinear()
+      .domain([0, max(data.map((d) => d[2]))])
```

Comment:
It looks like our data contains some negative values, unsure if those are sensor errors or something else but just to account for them maybe we should change this to:
`.domain([min(data.map((d) => d[2]).concat(0)), max(data.map((d) => d[2]))])`
What do you think?